### ISSUE TITLE
DeprecationWarning: node --debug is deprecated

### ISSUE THREAD

**Issue by Aditrc** [Microsoft/vscode#27704]
- VSCode Version: Code 1.12.2 (19222cdc84ce72202478ba1cec5cb557b71163de, 2017-05-10T13:20:36.315Z)
- OS Version: Windows_NT ia32 10.0.15063

when I debug Node.js v7.10.0, console shows:

```
检测到 Node.js v7.10.0，因此使用旧版协议调试。
node --debug-brk=11979 --nolazy run.js 
(node:16016) DeprecationWarning: node --debug is deprecated. Please use node --inspect instead.
Debugger listening on 127.0.0.1:11979
(node:16016) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: 没有给出操作
(node:16016) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by weinand** [comment]
@Aditrc add `"protocol": "inspector"` to your launch config and node-debug will pass the "--inspect" option to node.js.
(See https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_supported-nodelike-runtimes)

**Comment by Aditrc** [comment]
检测到 Node.js v7.10.0，因此使用旧版协议调试。
but why is the "v7.10.0" detected(检测到) and still using the old protocol(旧版协议)?

**Comment by weinand** [comment]
Because we made the decision that we use the new "inspector" protocol only for node "v8.x" because the new "inspector" protocol in node "v7" is still experimental:

```
sardinia:~ weinand$ node --version
v7.10.0
sardinia:~ weinand$ node --inspect
Debugger listening on port 9229.
Warning: This is an experimental feature and could change at any time.
```
So our approach is "conservative": as long as node.js issues the warning from above, we do not make the "inspect" protocol the default.

**Comment by GautamGupta26** [comment]
Was getting this on Node.js Version: v8.2.1.
This got solved on installing version v6.11.2.
